engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -90,6 +91,7 @@ public void initialise() {
         new WireframeTrigger(renderingDebugConfig, this);
 
         addDesiredStateChange(new BindFBO(READONLY_GBUFFER, displayResolutionDependentFBOs));
+        addDesiredStateChange(new SetRenderBufferMask(READONLY_GBUFFER, displayResolutionDependentFBOs, true, false, false));
```

Comment:
Can we change the signature here so that the mask (true, false false) is in the first three arguments of the constructor?

I'm also wondering if we should rename SetRenderBufferMask to SetFboWriteMask instead.